(Mostly) Automate data migration fixture import process (#1012)
pulibrary/pdc_describe: docs/test_data_in_production.md

These will need to be kept updated with changes to the UI, changes to the metadata schema, etc.

### 3. Create or refresh the work in staging
  1. Regenerate the json representations of the migration data by running:
     ```
     DATA_MIGRATION=true bundle exec rspec spec/system/data_migration
     ```

  2. Copy the newly created .json files to the server where you want to load them: 

>     
>     `scp tmp/data_migration/*.json deploy@p
> dc_describe_staging1.princeton.edu:/tmp`

Reviewer: ```suggestion
    ```
    scp tmp/data_migration/*.json deploy@pdc_describe_staging1.princeton.edu:/tmp
    ```
```

> 1. Then, ssh to the server where you want to create this sample work as the `deploy` user. 
> 2. Run the import rake task, specifying the location of the .json files and the netid of the user they should import as:

Reviewer: ```suggestion
3. Then, ssh to the server where you want to create this sample work as the `deploy` user. 
4. Run the import rake task, specifying the location of the .json files and the netid of the user they should import as:
```

  ```

>   bundle exec rake works:import_works\[/path/to/json/files,bs3097]

Reviewer: ```suggestion
  bundle exec rake works:import_works\[/tmp,bs3097]
```

In the previous step, I think you identify a destination, so I think you could use that same destination here... Except there may be other files you don't want in `/tmp` ... maybe a date-stamped subdirectory just for this transfer? Or maybe the script will skip files that aren't json?

Reviewer 2: The JSON files are not going to /tmp. They are going to a tmp/data_migration directory within the application. When they are copied to the server, they will go wherever the user decides to put them.
